File: Entity/Paciente.cs
using System;

namespace Entity
{
    public class Paciente
    {
        public string identificacion { get; set; }
        public double valorServicio { get; set; }
        public double tarifa { get; set; }
        public double salario { get; set; }
        public double copago { get; set; }

        public void CalcularCopago(){
            if(salario>2500000){
                tarifa=0.20;
                copago=tarifa*valorServicio;
            }else{
                if(salario<=2500000){
                    tarifa=0.10;
                    copago=tarifa*valorServicio;
                }
            }
        }
    }
}

File: APLICACIONES_SPADOTNET/ClientApp/src/app/liquidacion/modals/paciente.ts
export class Paciente {
    identificacion: string;
    valorServicio: number;
    salario:number;
    copago: number;
    Tarifa: number;


    CalcularCopago(): void{

        if(this.salario>2500000){

            this.Tarifa=0.20;
            this.copago=this.Tarifa*this.valorServicio;
        }else{
            this.Tarifa=0.10;
            this.copago=this.Tarifa*this.valorServicio;
        }
    }
}

